README.md

```diff
@@ -22,6 +59,14 @@ Released under the Apache License 2.0
 Ocean's [contributing guide](https://docs.ocean.dwavesys.com/en/stable/contributing.html)
 has guidelines for contributing to Ocean packages.
 
+This package uses `black` for python formatting. Before commiting changes run 
```

Comment:
Does it? Unless we enforce it in CI then I don't think we do. I also don't think we should.